router.js

```diff
@@ -0,0 +1,61 @@
+/**
+@license
+Copyright (c) 2018 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+*/
+
+/*
+  Basic router that calls a callback whenever the location is updated.
+
+  Sample use:
+  import { connect } from '../node_modules/@webcomponents/redux-helpers/connect-mixin.js';
+  import { installRouter } from '../node_modules/@webcomponents/redux-helpers/router.js';
+
+  import { navigate } from '../actions/app.js';
+
+  class MyElement extends connect(store)(HTMLElement) {
+    // ...
+
+    connectedCallback() {
+      // If you don’t have any other work to do other than dispatching an action,
+      // you can write something like:
+      installRouter(() => store.dispatch(navigate(window.location)));
```

Comment:
`installRouter` is actually generic enough to be used outside of custom elements. For example, in polymer-redux-hn, this is just called outside of the element at the top-level of hn-app.js.